packages/consumption/test/modules/notifications/NotificationEnd2End.test.ts

```diff
@@ -66,7 +66,7 @@ describe("End2End Notification via Messages", function () {
         TestNotificationItemProcessor.reset();
     });
 
-    test("sender: sent Notification", async function () {
+    test("sender: save Notification with status 'Sent'", async function () {
```

Comment:
I think you forgot to undo stuff here

Reply:
So I would change this to "sender:  mark LocalNotification as sent" in order to be consistent?

Reply:
no, `sender: sent Notification` was correct. this test tests the function sentNotification which sets the test name :)

Reply:
but I'm also ok with your new test name